### DISCUSSION TITLE
Is the new code search a work in progress

### DISCUSSION THREAD

**Discussion by ThereChave** [community/community#55639] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Is the new code search a work in progress? Previously there was an option to switch to the old code search, and now the new search is mandatory, but the new search faces many problems, and the developers seem to turn a blind eye to these problems
The search can only display 5 pages
Can't sort by time
The content that was searchable before can not be found at all in the new search
...
This makes the search difficult to use, and there are countless discussions like this. Does github launch this search in the hope of becoming more closed?

**Comment by DouYinApi** [comment]
Yes, we will be more closed, we don't care about user needs, we have to turn off old features, you can't control us
